### PR TITLE
Add formjs w/ git auto-update via single package.json

### PR DESCRIPTION
**Pull request by the94air**
Pull request for issue: #10710
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/the94air/formjs
 * Official website (optional, not the repository): https://form.js.org/
 * NPM package url (optional):
 * License and its reference: MIT [reference](https://github.com/the94air/formjs/blob/master/License.txt)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 1
   - Count of stars: 3
   - Count of forks: 0
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/formjs/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "formjs",
+  "version": "1.1.1",
```
Comment (pvnr0082t) [inline ajax/libs/formjs/package.json]: We don't need version field if add library via single package.json.

File: ajax/libs/formjs/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/the94air/formjs.git",
+    "fileMap": [{
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }]
```
Comment (pvnr0082t) [inline ajax/libs/formjs/package.json]: wrong format. Should be:
```
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/*"
        ]
      }
    ]
```
Thanks.

**Comment by PeterDaveHello** [comment]
@pvnr0082t wrong commit message and title.

**Comment by PeterDaveHello** [comment]
@pvnr0082t there is no need to cc author him/her self :)

**Comment by PeterDaveHello** [comment]
@pvnr0082t notice all the old versions can't be fetched.

**Comment by pvnr0082t** [comment]
@PeterDaveHello Oh, sorry I didn't notice. Thanks.

**Pull request merged**